docs/building_lbann.rst

```diff
@@ -39,8 +39,30 @@ Setup Spack
         export SPACK_ROOT=<path to installation>/spack.git
         source ${SPACK_ROOT}/share/spack/setup-env.sh
 
+2.  Update spack to use the new clingo concretizer.  There are two
+    steps to this: bootstrapping clingo and modifying the repositories
+    configuration file.
 
-2.  LBANN will use `Spack environments
+    .. code-block:: bash
+
+       module load gcc/8.3.1     # Load a compiler with C++17 support
+       spack compiler add        # Make Spack aware of the new compiler
+       spack solve zlib          # Force Spack to bootstrap clingo
+
+    Then open the `config.yaml` file:
+
+    .. code-block:: bash
+
+       emacs -nw <path to installation>/spack.git/etc/spack/config.yaml
+
+    Ensure the `config.yaml` file contains the following lines:
+
+    .. code-block:: bash
+
+       config:
+         concretizer: clingo
+
```

Comment:
We don't need to tell folks how to edit files. If they don't already know, they need to read a "linux command line 101" primer first, and that's not our job.
```suggestion
    Create the file `${SPACK_ROOT}/etc/spack/config.yaml` if it doesn't exist. Verify that this file contains the following lines, and add them if necessary:

    .. code-block:: bash

       config:
         concretizer: clingo

```